scss/main.scss

```diff
@@ -0,0 +1,38 @@
+@import "variables";
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  padding: 71px 20px 0 20px;
+}
+
+.creative-crew-title {
+  font-family: $first-font;
+  font-size: 36px;
+  height: 47px;
+  margin-bottom: 38px;
+  letter-spacing: -0.72px;
+  line-height: 48px;
+  width: 278px;
+}
+
+.we-are-title {
+  font-family: $second-font;
+  font-size: 18px;
+  height: 27px;
+  margin-bottom: 10px;
+  text-transform: uppercase;
+  width: 117px;
+}
+
+.information-team {
+  font-family: $second-font;
+  font-size: 14px;
+  height: 87px;
+  width: 333px;
+}
```

Comment:
This shouldn't be here since this PR is just a `Hello World`.

Reply:
551c90c